### DISCUSSION TITLE
cmd prompt closing/crashing

### DISCUSSION THREAD

**Discussion by PersonOnTheNet** [legendary-gl/legendary#263] [Q&A]
after i installed the legendary executable it ran and then closed after about 3 seconds, i don't think it installed or did anything, im talking about the file after installing the windows version of legendary to be a bit more specific, i just need to know how i can fix that, thanks in advance

**Comment by CommandMC** [comment]
https://github.com/derrod/legendary/wiki/Setup-Instructions is probably going to explain most of your questions

**Reply by PersonOnTheNet** [reply]
thank you, im going to try it
